README.md

```diff
@@ -6,16 +6,37 @@
 [![Build Status](https://ci.appveyor.com/api/projects/status/github/invenia/JLSO.jl?svg=true)](https://ci.appveyor.com/project/invenia/JLSO-jl)
 [![Codecov](https://codecov.io/gh/invenia/JLSO.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/invenia/JLSO.jl)
 
-JLSO is a storage container for serialized Julia objects.
+JLSO is a storage container for serialized Julia objects.  
```

Comment:
I'm guessing the `index.md` file should be using the README?

Reply:
I have often made that argument yes.
that the docs index.md and the readme should have the same info.

I have set it up before on some repos.